torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12298,8 +12300,24 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            assert_jit_shape_analysis=True,
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
-           sample_inputs_func=sample_inputs_permute,
+           sample_inputs_func=partial(sample_inputs_permute, is_method=False),
            reference_inputs_func=reference_inputs_permute),
+    OpInfo('permute',
+           op=lambda x, *dims: x.permute(*dims),
+           variant_test_name='tensor_method',
+           ref=np.transpose,
```

Comment:
While this is moot if we decide not to add this OpInfo variant, I don't think `np.transpose` is a valid reference implementation for `torch.Tensor.transpose` because I believe `np.transpose` doesn't accept varargs. I think this NumPy reference would have to be `ndarray.transpose`?

Reply:
Fair enough.